dbt_copilot_helper/utils/application.py

```diff
@@ -1,13 +1,82 @@
+import json
 from pathlib import Path
+from typing import Dict
 
+import boto3
 import yaml
 from yaml.parser import ParserError
 
+from dbt_copilot_helper.utils.aws import get_aws_session_or_abort
+from dbt_copilot_helper.utils.aws import get_profile_name_from_account_id
 from dbt_copilot_helper.utils.files import load_and_validate_config
 from dbt_copilot_helper.utils.messages import abort_with_error
 from dbt_copilot_helper.utils.validation import BOOTSTRAP_SCHEMA
 
 
+class Environment:
+    name: str
+    account_id: str
+    sessions: Dict[str, boto3.Session]
+
+    def __init__(self, name: str, account_id: str, sessions: Dict[str, boto3.Session]):
+        self.name = name
+        self.account_id = account_id
+        self.sessions = sessions
+
+    @property
+    def session(self):
+        if self.account_id not in self.sessions:
+            self.sessions[self.account_id] = get_aws_session_or_abort(
+                get_profile_name_from_account_id(self.account_id),
+            )
+
+        return self.sessions[self.account_id]
+
+
+class Application:
```

Comment:
Is it worth adding docstrings to these two classes? Maybe saying how they map or don't map onto AWS concepts

Reply:
Probably not worth the additional maintenance. I'd be interested to understand what is missing in code and accompanying tests that might help convey how these classes apply to copilot.